internal/iso/iso.go

```diff
@@ -122,60 +117,82 @@ func (h *Handler) RoundTrip(req *http.Request) (*http.Response, error) {
 	// RoundTripper needs a Transport to execute a HTTP transaction
 	// For our use case the default transport will suffice.
 	resp, err := http.DefaultTransport.RoundTrip(req)
-	// resp, err := h.RoundTripper.RoundTrip(req)
 	if err != nil {
-		h.Logger.Info("HTTP request didn't receive a response", "sourceIso", h.SourceISO, "error", err)
+		log.Info("issue getting the source ISO", "sourceIso", h.SourceISO, "error", err)
 		return nil, err
 	}
+	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusPartialContent {
+		log.Info("the request to get the source ISO returned a status other than ok (200)", "sourceIso", h.SourceISO, "status", resp.Status)
+		return resp, nil
+	}
+	// by setting this header we are telling the logging middleware to not log its default log message.
+	// we do this because there are a lot of partial content requests.
+	resp.Header.Set("X-Global-Logging", "false")
 
 	if req.Method == http.MethodHead {
-		// Fuse client typically make a HEAD request before they start requesting content.
-		h.Logger.V(1).Info("HTTP HEAD request received, patching only occurs on 206 requests")
+		// Fuse clients typically make a HEAD request before they start requesting content.
+		log.Info("HTTP HEAD method received")
```

Comment:
Should this be debug or provides any value as is?

Reply:
I'm undecided on this at the moment. From testing with just one physical machine, there's about 2-3 HEAD requests that are made and it was helpful to show that progress was being made.